CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Blip 
 
-## [Unreleased]
+## [0.6.1] - 2019-08-14
```

Comment:
should we change the date here?

Reply:
@BigFab For sure this date will have to be changed. 
That will be part of the final bumping. 
We have to integrate as well the viz and tideline releases in package.json